createFolders.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(readlink "$0")")"
+
+if [[ $# -lt 1 ]]; then
+  echo "Usage: bash createFolders.sh <programming language>" >&2
+  exit 1
+fi
+
+programmingLanguage=$1
+mkdir "${DIRECTORY}/${programmingLanguage}"
+rsync --archive --include='*/' --exclude='*' "${DIRECTORY}/Python/" "${DIRECTORY}/${programmingLanguage}"
```

Comment:
nit: break long lines for better readability.

```suggestion
rsync --archive --include='*/' --exclude='*' "${DIRECTORY}/Python/" \
    "${DIRECTORY}/${programmingLanguage}"
```